requirements.moreh.txt

```diff
@@ -0,0 +1,19 @@
+transformers==4.29.2
+datasets==2.14.6
+evaluate==0.4.1
+peft==0.5.0 
+scikit-learn==1.3.2
+loralib==0.1.2
+tqdm==4.66.1
+diffusers==0.21.4
+Pillow==10.1.0
+huggingface_hub==0.18.0
+accelerate==0.24.0
+mlflow==2.9.1
```

Comment:
should be 2.11.1 https://github.com/moreh-dev/moreh-testbench/pull/143

Reply:
As discussed with Loc, we believe it's better to use the latest MLflow version.